.github/workflows/ci-restart-failed-daily-conda-builds.yml

```diff
@@ -0,0 +1,33 @@
+name: Restart failed daily conda builds
+
+on:
+  workflow_run:
+    workflows: [ci-ifcopenshell-conda-daily]
+    types:
+      - completed
+
+env:
+  REPO_OWNER: IfcOpenShell
+  MY_WORKFLOW: ci-ifcopenshell-conda-daily
+
+jobs:
+  restart-failed-runs:
+    runs-on: ubuntu-latest
+    if: ${{ github.event.workflow_run.conclusion == 'failure' }}
+    steps:
+    - name: Check for failed runs
+      id: check-failed-runs
+      run: |
+        echo "Checking for failed runs..."
+        # Replace "my-workflow" with the name of your workflow
```

Comment:
Remove I guess

Reply:
Regarding your comment on recursiveness. You are right. If the ci job for some reason continues to produce the same reoccurring 143 error this will run forever. 

How about I comment out the "Restart failed runs" and let it run the "check-failed-runs" only for a while to ensure that it's safe before we use it properly? That way this ci only runs once and it's more in "monitoring mode" only for a while to see that it reaches the right conclusions in the daily jobs.

To be honest I haven't really gotten to test this ci workflow properly yet since the runs on my test repo never (or rarely) manages to run into the 143 error :( But the daily ci conda build seems to always run into arbitrary 143 errors. So I thought it might be a good idea to test this out here? 

While we run this in a "monitoring mode" I can find a way to stop any recursiveness by ensuring that any ci build is only retried once?